CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ return values of avifImageCopy() and avifImageAllocatePlanes().
   and avifRGBImage::avoidLibYUV to AVIF_TRUE.
 * avifRGBImage::chromaUpsampling now only applies to conversions that need
   upsampling chroma from 4:2:0 or 4:2:2 and has no impact on the use of libyuv.
+  Set avifRGBImage::avoidLibYUV accordingly for that.
```

Comment:
Nit: "for that" is not clear. How about "to control the use of libyuv"?

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1131